### PR TITLE
SC-8175 - Experience Element Timing docs

### PR DESCRIPTION
**Pull request by gr0**
Hi guys. Please have a look at the Element Timing docs for experience. The text is in the repo and it looks as follows when rendered:

element timing docs

### CODE REVIEW

File: docs/experience/element-timing.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+/>
+
+## Element Timing Overview
+
+When accessing the Element Timing from the left menu of your Experience App you will see numerous metrics related to the annotated elements of your web application:
+
+* Total number of annotated element loads.
+* 90th percentile of annotated elements render time in milliseconds.
+* 95th percentile of annotated elements load time in milliseconds.
+* A histogram of average load and render time of annotated elements.
```
Comment (otisg) [inline docs/experience/element-timing.md:40]: histogram of average load?  Is this correct?  Should't it just be a histogram of load and render time for annotated elements?  Not avg load.

File: docs/experience/element-timing.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+<img
+  class="content-modal-image"
+  alt="Element Timing Overview"
+  src="../../images/experience/elementtiming/element_timing_overview.png"
+  title="Element Timing Overview"
+/>
+
+## Element Timing Details
+
+Accessing the details of an annotated web page element is as simple as chosing the interesting name from the Top Element Timings table in the Element Timing Overview.
```
Comment (otisg) [inline docs/experience/element-timing.md:53]: chosing => choosing
interesting name -- maybe you mean something like "element of interest"?

File: docs/experience/element-timing.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+The Element Timing Overview inlcudes timing information for each type of annotated web page element which include:
```
Comment (otisg) [inline docs/experience/element-timing.md:55]: inlcudes  => includes
which include - zap, no need

File: docs/experience/element-timing.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+ 
+* Number of loads for a single annotated element.
+* 90th percentile of annotated element render time in milliseconds.
+* 95th percentile of annotated element render time in milliseconds.
+* A histogram of average load and render time of annotated element.
```
Comment (otisg) [inline docs/experience/element-timing.md:60]: Same comment as above

File: docs/experience/element-timing.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
+title: Monitoring Browser Element Timing
+description: Gathering information about annotated elements on the web page
+
+Experience allows using the Element Timing API to annotate the critical elements of your site. While knowing the rendering time of the key elements of the page is not enough for measuring satisfaction of the page load, it is crucial to know when they are displayed on the screen.
```
[the remark this answers is not in the document]
Comment (gr0) [reply docs/experience/element-timing.md]: Adjusted.

File: docs/experience/element-timing.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
+title: Monitoring Browser Element Timing
+description: Gathering information about annotated elements on the web page
+
+Experience allows using the Element Timing API to annotate the critical elements of your site. While knowing the rendering time of the key elements of the page is not enough for measuring satisfaction of the page load, it is crucial to know when they are displayed on the screen. 
+
+## Supported Page Elements
+
+The Element Timing API is only supported by Google Chrome and works on the following page elements:
```
[the remark this answers is not in the document]
Comment (gr0) [reply docs/experience/element-timing.md:8]: Adjusted.

File: docs/experience/element-timing.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+
+* `<img>` elements
+* `<image>` elements inside an `<svg>` tag
+* The poster images of the `<video>` elements
+* Elements with `background-image` property
+* Text nodes
+
+## Annotating Page Elements
+
+To include timing data for your web page elements you must annotate them with the `elementtiming` property and give the element a name. The provided name will be associated with the element timing when viewed in Experience dashboards. You can have as many `elementtiming` attributes as you want, but keep in mind that this should be added to the crucial elements on your site, the one which reflects the key elements your users should see, so that you can see how fast they actually get them rendered. For example, if you would like to know the timing information associated with the featured image of your web page you could have your `<img>` tag annotated as follows:
```
[the remark this answers is not in the document]
Comment (gr0) [reply docs/experience/element-timing.md]: Adjusted.

File: docs/experience/element-timing.md (hunk 1 above)
Comment (gr0) [reply docs/experience/element-timing.md:40]: Adjusted.

File: docs/experience/element-timing.md (hunk 2 above)
Comment (gr0) [reply docs/experience/element-timing.md:53]: Adjusted. Thanks!

File: docs/experience/element-timing.md (hunk 3 above)
Comment (gr0) [reply docs/experience/element-timing.md:55]: Adjusted, thanks!

File: docs/experience/element-timing.md (hunk 4 above)
Comment (gr0) [reply docs/experience/element-timing.md:60]: Thanks!

**Comment by amir-hadzic** [comment]
This looks good @gr0. I would suggest few changes to the annotation part to make this read more like a tutorial on getting started. We will be linking to this page when user has no tracked elements, so the focus should be on getting started.

docs-review

I suggest moving the second paragraph up and renaming it to Getting Started. Also I would split the text there in 2-3 paragraphs.

**Comment by gr0** [comment]
@amir-hadzic Good idea, will adjust.

**Comment by gr0** [comment]
> This looks good @gr0. I would suggest few changes to the annotation part to make this read more like a tutorial on getting started. We will be linking to this page when user has no tracked elements, so the focus should be on getting started.

Done :) 

> I suggest moving the second paragraph up and renaming it to Getting Started. Also I would split the text there in 2-3 paragraphs.

Done :)

File: docs/experience/element-timing.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
+title: Monitoring Browser Element Timing
+description: Gathering information about annotated elements on the web page
+
+Experience uses the Element Timing API to annotate the critical elements of your site. While knowing the rendering time of the key elements of a page is not enough to fully capture user satisfaction, it is crucial to know how long it takes for them to be displayed on the screen.
```
Comment (adnanrahic) [inline docs/experience/element-timing.md:4]: Experience uses the Element Timing API to annotate the critical elements of your site. While knowing the rendering time of the key elements of a page is not enough to fully capture user satisfaction, it is crucial to know how long it takes for them to be displayed on the screen.
=>
Experience uses the Element Timing API to annotate critical elements on your website/webapp. Knowing the rendering time of the key elements on a page is not enough to fully capture user satisfaction, but it is crucial to know how long it takes for them to be displayed on the screen.

Comment (gr0) [reply docs/experience/element-timing.md]: Done, thanks!

**Comment by gr0** [comment]
Yup, everything was adjusted. Thank you for review guys!

**Pull request merged**